app/operations/project_create.rb

```diff
@@ -22,6 +22,9 @@ def create_project_from_request(request)
       project.draft_doi
       project.save!
 
+      # Link the request to the project
```

Comment:
@hectorcorrea should there be a request.save here so the request is saved before any error occurs?